compiler/src/dotty/tools/dotc/util/Spans.scala

```diff
@@ -59,6 +59,9 @@ object Spans {
       if (poff == SyntheticPointDelta) start else start + poff
     }
 
+    def pointMayBeIncorrect =
+      pointDelta == 0 && end - start >= SyntheticPointDelta
```

Comment:
Is it dubious to make this API of span? Why this check on pointDelta if any large span may make it overflow?

Reply:
If pointDelta it too large, pointDelta will be set to 0, so only then it can be wrong.

Reply:
I withdraw my objection to adding the method to Span. As a footnote, there is no range check on point, so "incorrect" code can attempt to assign a point "outside" the span. This was an ongoing issue in Scala 2. I haven't looked again yet at the "multiple vars" position issue, but that is the sort of incorrect code I mean.